offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/projects/ProjectMainConnector.groovy

```diff
@@ -163,7 +166,18 @@ class ProjectMainConnector implements CreateProjectDataSource, CreateProjectSpac
       }
   
       return projectDbConnector.addProjectAndConnectPersonsInUserDB(projectIdentifier, projectApplication)
-  
-      return new Project(projectIdentifier, projectTitle, projectApplication.getLinkedOffer())
     }
+    
+  private void handleOperations(IOperation operation) {
+    IApplicationServerApi api = openbisClient.getV3()
+    
+    SynchronousOperationExecutionOptions options = new SynchronousOperationExecutionOptions()
+    List<IOperation> ops = Arrays.asList(operation)
```

Comment:
```suggestion
    SynchronousOperationExecutionOptions execitionOptions = new SynchronousOperationExecutionOptions()
    List<IOperation> operationOptions = Arrays.asList(operation)
```

Reply:
Not required changes but it would help to better understand what these options are.

Reply:
When accepting this the code down below needs also to be adjusted